src/FindNetworkEndpoint/index.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { FindNetworkEndpoint } from './index';
+
+describe('FindNetworkEndpoint', () => {
+  describe('has two different lengths of arrays', () => {
+    it('throw error', () => {
+      const startNodeId = 1;
+      const fromIds = [1, 2];
+      const toIds = [1];
+      const compute = new FindNetworkEndpoint(startNodeId, fromIds, toIds);
+      expect(() => compute.find()).toThrowError(
+        `${fromIds} and ${toIds} must have the same length`
+      );
+    });
+  });
+
+  describe('has two same lengths of arrays', () => {
+    describe('starting node is not existing in fromIds', () => {
+      it('throw error', () => {
+        const startNodeId = 10;
+        const fromIds = [1, 2];
+        const toIds = [5, 7];
+        const compute = new FindNetworkEndpoint(startNodeId, fromIds, toIds);
```

Comment:
It shouldn't thorw is should just return the starting node as the graph is just single node

Reply:
it should throw error as the starting node is not within the network.